src/Http.ts

```diff
@@ -14,27 +13,24 @@ export default class Http {
     this.axios = Axios.create({
       baseURL: this.host + 'api/v2/storefront',
       headers: {
-        'Content-Type': 'application/vnd.api+json'
+        'Content-Type': 'application/json'
       },
       paramsSerializer: (params) => {
         return qs.stringify(params, { arrayFormat: 'brackets' })
       }
     })
   }
 
-  protected async spreeResponse(method: string, route: string, params: any = {}) {
-    if (this.spreeTokens) {
-      this.setHeaders()
```

Comment:
@arudnicka don't change anything here. You just need to reset old `Bearer` token if it doesn't pass to the method.

Reply:
please add method `removeHeaders()` and put it into `else` statement

Reply:
This is only temporary solutions. Your idea is good, but it needs more work.